modules/oc-mirror-imageset-config-parameters-v2.adoc

```diff
@@ -0,0 +1,395 @@
+
+.`DeleteImageSetConfiguration` parameters
+[cols="2,2a,1a",options="header"]
+|===
+|Parameter
+|Description
+|Values
+
+|`apiVersion`
+|The API version for the `DeleteImageSetConfiguration` content.
+|String. For example: `mirror.openshift.io/v2alpha1`.
+
+|`delete`
+|The configuration of the image set to delete.
+|Object
+
+|`delete.additionalImages`
+|The additional images configuration of the delete image set.
+|Array of objects. For example:
+[source,yaml]
+----
+additionalImages:
+  - name: registry.redhat.io/ubi8/ubi:latest
+----
+
+|`delete.additionalImages.name`
+|The tag or digest of the image to delete.
+|String. For example: `registry.redhat.io/ubi8/ubi:latest`
+
+|`delete.operators`
+|The Operators configuration of the delete image set.
+|Array of objects. For example:
+[source,yaml]
+----
+operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:vBranch Build
+    packages:
+      - name: elasticsearch-operator
+        minVersion: '2.4.0'
+----
+
+|`delete.operators.catalog`
+|The Operator catalog to include in the delete image set.
+|String. For example: `registry.redhat.io/redhat/redhat-operator-index:v4.15`.
+
+|`delete.operators.full`
+|When true, deletes the full catalog, Operator package, or Operator channel.
+|Boolean. The default value is `false`.
+
+|`delete.operators.packages`
+|The Operator packages configuration.
+|Array of objects. For example:
+[source,yaml]
+----
+operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:vBranch Build
+    packages:
+      - name: elasticsearch-operator
+        minVersion: '5.2.3-31'
+----
+
+|`delete.operators.packages.name`
+|The Operator package name to include in the delete image set
+|String. For example: `elasticsearch-operator`.
+
+|`delete.operators.packages.channels`
+|The Operator package channel configuration.
+|Object
+
+|`delete.operators.packages.channels.name`
+|The Operator channel name, unique within a package, to include in the delete image set.
+|String. For example: `fast` or `stable-v4.15`.
+
+|`delete.operators.packages.channels.maxVersion`
+|The highest version of the Operator to delete within the selected channel.
+|String. For example: `5.2.3-31`
+
+|`delete.operators.packages.channels.minVersion`
+|The lowest version of the Operator to delete within the selected in which it exists. 
+|String. For example: `5.2.3-31`
+
+|`delete.operators.packages.maxVersion`
+|The highest version of the Operator to delete across all channels in which it exists. See the following note for further information.
+|String. For example: `5.2.3-31`.
+
+|`delete.operators.packages.minVersion`
+|The lowest version of the Operator to delete across all channels in which it exists. See the following note for further information.
+|String. For example: `5.2.3-31`.
+
+|`delete.operators.packages.bundles`
+|The selected bundles configuration.
+|Array of objects.
+
+You cannot choose both channels and bundles for the same operator.
```

Comment:
Should you add that as a note ? Also not sure of the context on why it has been added in the imageSetConfiguration of delete alone ?